web/src/components/repo/pipeline/PipelineStepList.vue

```diff
@@ -10,21 +10,29 @@
         </div>
         <span>{{ pipeline.author }}</span>
       </div>
-      <div class="flex space-x-1 items-center min-w-0">
+      <a
+        v-if="pipeline.event === 'pull_request'"
+        class="flex items-center space-x-1 text-wp-link-100 hover:text-wp-link-200 min-w-0"
+        :href="pipeline.link_url"
+        target="_blank"
```

Comment:
Do we need to use `target="_blank"`? As a user I would prefer to control this on my own.

Reply:
Thanks for removing it. Looks like we use it all over the place. Might need a discussion if we want to use it or not.